_includes/cockroachcloud/planning-your-cluster.md

```diff
@@ -0,0 +1,17 @@
+### Requirements
+
+- Multi-region clusters require at least 3 nodes per region because 2-replica configurations are less reliable than a single replica.
+- We don't allow 2-region clusters because they are less stable and would be unable to survive a single region failure.
+- You can have a maximum of 9 regions per cluster through the console. If you need to add more regions, [contact us](https://support.cockroachlabs.com).
```

Comment:
Do we refer to the CC console just as the "console"? I see some places where we call it "CockroachCloud Console" and others as the "Console."

So maybe first instance on a page, call it out as CockroachCloud Console, and thereafter just Console.